shipgen/MuonBackGenerator.cxx

```diff
@@ -165,10 +165,13 @@ Bool_t MuonBackGenerator::ReadEvent(FairPrimaryGenerator* cpg)
   }
   if (fPhiRandomize){phi = gRandom->Uniform(0.,2.) * TMath::Pi();}
   if (fsmearBeam > 0) {
-     Double_t r = fsmearBeam + 0.8 * gRandom->Gaus();
-     Double_t _phi = gRandom->Uniform(0., 2.) * TMath::Pi();
-     dx = r * TMath::Cos(_phi);
-     dy = r * TMath::Sin(_phi);
+      Double_t x = gRandom->Gaus(0, 0.8);
+      Double_t y = gRandom->Gaus(0, 0.8);
```

Comment:
Why introduce these intermediate variables, if you only use them to assign to dx and dy?